### DISCUSSION TITLE
Downloads not in default directory

### DISCUSSION THREAD

**Discussion by pummra** [cross-seed/cross-seed#96] [General]
I have cross-seed up and running and working well. At the moment I have it running manually but the ambition is to set it up to be automated. 

I am running this on my seedbox and I have autotools moving my completed downloads in to folders based on their labels (which are added my Radarr and Sonarr). 

When I run cross-seed at the moment I have it putting the torrent files in to a watch folder and I am then manually having to sell each torrent where the correct files are to begin seeding from. Is there any way that I can automate this so that it looks in the right place straight away and would allow for greater automation?

**Comment by afraazali** [comment]
If you’re using rtorrent, I think the wiki has documentation on how to setup cross-seed in daemon mode.

Personally I’m using qbit, and run the script on cron. Not ideal, but it gets the job done.

**Reply by pummra** [reply]
I am happy getting it set up in daemon mode, I more concerned that if the downloads are auto moved out of the default download directory cross-seed will just end up downloading the file again.

**Reply by afraazali** [reply]
With daemon mode, you can have rtorrent make a request to a webhook that cross-seed expose. Conventionally that request would be made on torrent completion, but you might be able to make that request on a different event, perhaps **after** autotools does its move? Alternatively, is using hardlinks an option for you?

**Reply by pummra** [reply]
Hardlinks. That is a good idea. Pretty certain that is an option in autotools.

**Reply by mmgoodnow** [reply]
With direct injection cross-seed will look up the original torrent in your client and mirror its data location, so I think this should be a non issue.

**Reply by pummra** [reply]
Even better. I will give that a go now.

**Reply by ajhorvath20** [reply]
> With direct injection cross-seed will look up the original torrent in your client and mirror its data location, so I think this should be a non issue.

Hi, so currently I am running `daemon `mode and every time file complete `cross-seed` look for similar torrents but I am not sure if it checks all the existing torrents too or only the completed torrent. Because in  `search `mode I had set it to `--exclude` which search for torrents up to 5 days because trackers have the same torrent file 2-3 days later. So how `daemon `mode work in this scenario. Thanks
